data/org.freedesktop.portal.InputCapture.xml

```diff
@@ -0,0 +1,552 @@
+      <arg type="aa{sv}" name="barriers" direction="in"/>
+      <arg type="u" name="serial" direction="in"/>
+      <arg type="o" name="handle" direction="out"/>
+    </method>
+
+    <!--
+        Enable:
+        @session_handle: Object path for the #org.freedesktop.portal.Session object
+        @options: Vardict with optional further information
+        @handle: Object path for the #org.freedesktop.portal.Request object representing this call
+
+        Enable input capturing. This does not immediately trigger capture, it
+        merely enables the capturing to be triggered at some future point
+        (e.g. by the cursor moving across a barrier). If and when that happens,
+        the #org.freedesktop.portal.InputCapture::Activated signal is emitted.
+
+        Supported keys in the @options vardict include:
+        <variablelist>
+          <varlistentry>
+            <term>handle_token s</term>
+            <listitem><para>
+              A string that will be used as the last element of the @handle. Must be a valid
+              object path element. See the #org.freedesktop.portal.Request documentation for
+              more information about the @handle.
+            </para></listitem>
+          </varlistentry>
+        </variablelist>
+    -->
+    <method name="Enable">
+      <arg type="o" name="session_handle" direction="in"/>
+      <arg type="a{sv}" name="options" direction="in"/>
+      <arg type="o" name="handle" direction="out"/>
+    </method>
+
+    <!--
+        Disable:
+        @session_handle: Object path for the #org.freedesktop.portal.Session object
+        @options: Vardict with optional further information
+        @handle: Object path for the #org.freedesktop.portal.Request object representing this call
+
+        Disable input capturing. No
+        #org.freedesktop.portal.InputCapture::Disabled signal will be emitted.
+
+        If input capturing is currently ongoing, no
+        #org.freedesktop.portal.InputCapture::Deactivated signal will be
+        emitted.
+
+        Input events will not be captured until a subsequent
+        org.freedesktop.portal.InputCapture.Enable() call.
+    -->
+    <method name="Disable">
+      <arg type="o" name="session_handle" direction="in"/>
+      <arg type="a{sv}" name="options" direction="in"/>
+      <arg type="o" name="handle" direction="out"/>
+    </method>
+
+    <!--
+        Release:
+        @session_handle: Object path for the #org.freedesktop.portal.Session object
+        @options: Vardict with optional further information
+        @handle: Object path for the #org.freedesktop.portal.Request object representing this call
+
+        Release any ongoing input capture. No
+        #org.freedesktop.portal.InputCapture::Deactivated signal will be emitted.
+
+        Supported keys in the @options vardict include:
+        <variablelist>
+          <varlistentry>
+            <term>cursor_position ii</term>
```

Comment:
Maybe make this `(dd)` to not loose precision.

Reply:
done (for all occurances)